assets/js/editor-components/search-list-control/types.ts

```diff
@@ -6,11 +6,11 @@ import type { ReactNode } from 'react';
 export type SearchListItemType = {
 	id: string | number;
 	name: string;
-	value: string;
-	parent: number;
-	count: number;
-	children: SearchListItemsType;
-	breadcrumbs: string[];
+	value?: string;
```

Comment:
We don't always provide the list with all properties to the `<SearchListControl>` and we don't need to. For example: https://github.com/woocommerce/woocommerce-blocks/blob/fix/6496/assets/js/blocks/attribute-filter/edit.tsx#L140-L148.

But this is not the complete solution because I didn't review the usage of `<SearchListControl>` in other places. I will revert this back to the original state. We should review the data structure of `<SearchListControl>` in a separate PR.

Reply:
Created https://github.com/woocommerce/woocommerce-blocks/issues/6578 for this.